app/assets/stylesheets/editor/_editor.scss

```
@@ -123,6 +124,54 @@ $top-height: 4rem;
   }
 }
 
+.editor-hidden-item-indicator {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+  padding: 0.125rem 0.5rem;
+  font-size: 0.75rem;
+  border-radius: 1rem;
+  background-color: var(--secondary);
+  color: #fff;
+  opacity: 0.5;
+  transition: opacity 200ms linear;
+  user-select: none;
+
+  &__tooltip {
+    $tooltip-bg-color: $bg-darker;
+    $tooltip-arrow-size: 0.5rem;
+
+    position: absolute;
+    top: calc(100% + #{$tooltip-arrow-size});
+    right: 0;
+    width: max-content;
+    max-width: 200px;
+    padding: 0.25em 0.5em;
+    background: $tooltip-bg-color;
+    pointer-events: none;
+    display: none;
+
+    &::before {
+      content: "";
+      position: absolute;
+      width: 0;
+      height: 0;
+      top: -$tooltip-arrow-size;
+      right: 0.5rem;
+      border-inline: $tooltip-arrow-size solid transparent;
+      border-bottom: $tooltip-arrow-size solid $tooltip-bg-color;
+    }
+  }
+
+  &:hover {
+    opacity: 1;
+  }
+
+  &:hover &__tooltip {
+    display: initial;
+  }
+}
```

Using &__ makes it a little hard to search for, I would prefer them separated

```suggestion
.editor-hidden-item-indicator {
  position: absolute;
  top: 1rem;
  right: 1rem;
  padding: 0.125rem 0.5rem;
  font-size: 0.75rem;
  border-radius: 1rem;
  background-color: var(--secondary);
  color: #fff;
  opacity: 0.5;
  transition: opacity 200ms linear;
  user-select: none;
  
  &:hover {
    opacity: 1;
  }
}

.editor-hidden-item-indicator __tooltip {
  $tooltip-bg-color: $bg-darker;
  $tooltip-arrow-size: 0.5rem;

  position: absolute;
  top: calc(100% + #{$tooltip-arrow-size});
  right: 0;
  width: max-content;
  max-width: 200px;
  padding: 0.25em 0.5em;
  background: $tooltip-bg-color;
  pointer-events: none;
  display: none;
  
  .editor-hidden-item-indicator __tooltip:hover & {
    display: initial;
  }

  &::before {
    content: "";
    position: absolute;
    width: 0;
    height: 0;
    top: -$tooltip-arrow-size;
    right: 0.5rem;
    border-inline: $tooltip-arrow-size solid transparent;
    border-bottom: $tooltip-arrow-size solid $tooltip-bg-color;
  }
}
```

---

Alright. Is `&--` okay though? I see one right below this, in `.drag-handle`

---

Yeah, I do use &--, which is a bit inconsistent, but it's convenience when adding it to stacks like `&:hover, &:active, &--active`

---

Did it myself instead of applying the suggestion because the suggestion seemed a little off. Lmk if I missed something.

---

That will do!